README.md

```diff
@@ -1,3 +1,17 @@
 # Pulsar Edit
 
 This is the website repo for Pulsar Edit. It contains our documenation and other information users might be looking for. Visit the [Website](https://pulsar-edit.github.io).
+
+
+## Guide
+
+### Tools Needed:
```

Comment:
```suggestion
### Recommended Tools:
```

Not necessary just thought it sounded better and fit with the narrative below more.